edb/edgeql-rust/src/tokenizer.rs

```diff
@@ -654,7 +654,15 @@ impl<'a> From<SpannedToken<'a>> for CowToken<'a> {
     }
 }
 
-fn unquote_bytes<'a>(s: &'a str) -> Result<Vec<u8>, String> {
+fn unquote_bytes<'a>(value: &'a str) -> Result<Vec<u8>, String> {
+    if value.starts_with("rb") || value.starts_with("br") {
```

Comment:
Let's do something like this:
```
let idx = value.find(|c| c == '\'' || c == '"').ok_or_else(...)?;
let prefix = value[..idx]
match prefix {
    "rb" | "br" => 
    "b" => 
    _ => Err(format!("unimplemented prefix {:?}", prefix)),
}
```
I think this would make potential new syntaxes safer.

Reply:
changed